src/hu/elte/txtuml/stdlib/util/CalendarInteface.java

```diff
@@ -0,0 +1,38 @@
+package hu.elte.txtuml.stdlib;
+import hu.elte.txtuml.api.model.external.ExternalClass;
+
+
+public interface CalendarInteface extends ExternalClass {
+	/*Following is the declaration for java.util.Calendar.add() method*/
+
+public void add(int field, int amount) ;
+
+
+
+/* declaration for java.util.Calendar.after() method */
+public boolean after(Object when) ;

```

Comment:
This function should take a `Calendar` parameter since `Object` is not a txtUML class nor an external class.